nf_core/components/components_utils.py

```diff
@@ -165,9 +165,10 @@ def get_components_to_install(subworkflow_dir: Union[str, Path]) -> Tuple[List[s
     return modules, subworkflows
 
 
-def get_biotools_id(tool_name) -> str:
+def get_biotools_response(tool_name: str) -> Optional[dict]:
```

Comment:
```suggestion
def get_biotools_response(tool_name: str) -> Optional[Dict]:
```
we are still on python 3.8... for now